Gruntfile.js

```diff
@@ -1,13 +1,87 @@
 module.exports = function(grunt) {
-    // Project configuration
+
     'use strict';
-    grunt.loadNpmTasks('grunt-contrib-requirejs');
-    grunt.loadNpmTasks('grunt-filerev');
-    grunt.loadNpmTasks('grunt-regex-replace');
-    grunt.loadNpmTasks('grunt-karma');
-    grunt.loadNpmTasks('grunt-coveralls');
-    grunt.loadNpmTasks('grunt-jasmine-nodejs');
-    grunt.loadNpmTasks('grunt-contrib-uglify-es');
+
+    require('load-grunt-tasks')(grunt);
+    grunt.loadNpmTasks('@lodder/grunt-postcss');
+
+    // files required for the 'page.html' template
+    const kbaseStaticDir = 'kbase-extension/static/kbase/',
+
+    staticFilesNoConcat = [
+        kbaseStaticDir + 'custom/*.css',
+        kbaseStaticDir + 'css/*.css',
+        '!' + kbaseStaticDir + 'css/*_concat.css',
+    ],
```

Comment:
this is used by stylelint -- it should run on css files that match these expressions